graphql_compiler/schema_generation/sqlalchemy/schema_graph_builder.py

```diff
@@ -65,3 +68,42 @@ def _get_edge_type_from_direct_edge(edge_name, direct_edge_descriptor):
     return EdgeType(edge_name, False, {}, {},
                     base_in_connection=direct_edge_descriptor.from_vertex,
                     base_out_connection=direct_edge_descriptor.to_vertex)
+
+
+def _get_sqlalchemy_indexes(vertex_name_to_table, vertex_types):
+    """Return the IndexDefinition objects corresponding to the indexes in the SQL tables."""
+    # TODO: Add indexes that do not correspond to neither the primary key nor the SQLAlchemy Table's
+    #       uniqueness constraints.
+    index_definitions = set()
+    for vertex_name, table in vertex_name_to_table.items():
+        for constraint in table.constraints:
+            if isinstance(constraint, (PrimaryKeyConstraint, UniqueConstraint)):
+                column_names = frozenset(column.name for column in constraint.columns)
+
+                # We ignore columns that have a SQL type with no matching GraphQL type.
+                invalid_columns = False
+                for name in column_names:
+                    if name not in vertex_types[vertex_name].properties:
+                        invalid_columns = True
+                        continue
+
+                if invalid_columns:
+                    continue
+
+                # Some SQL backends allow duplicate nulls in columns with unique indexes.
+                # However, other backends do not. Therefore, we set ignore_nulls=True to indicate
+                # that the backend may have duplicate nulls.
```

Comment:
This would take me 2 hours to implement. Basically I would have to:
1. For each database, I would have to google this information.
2. Create a map of dialect to `ignored_nulls` value.
3. Figure out the behavior of what to do if the dialect is not in the list I included. (We also have to make sure this list is kept up-to-date if SQLAlchemy ever includes a new dialect). 

Since `ignore_nulls` does not have any use in the compiler I don't think this is worth it. Do you think it's worth it?